docs/components/webapp.rst

```diff
@@ -43,6 +43,53 @@ Then you can start and stop the webapp, adjust files, and debug.
 FIXME(willkg): Review everything after this line.
 
 
+Setting up authentication and a superuser
+=========================================
+
+Creating a superuser
+--------------------
+
+If you want to do anything in the webapp admin, you'll need to create a
+superuser.
+
+Run this::
+
+  $ docker-compose run webapp python manage.py createsuperuser
+
+
+Then follow the prompts. The username and password don't matter at all, but the
+email address needs to match what you plan to use for Google auth.
+
+You can do this as many times as you like.
+
+
+Setting up the webapp for Google signin
+---------------------------------------
+
+In order to authenticate in the webapp using the "Google signin" button, you
+need to get a set of oauth credentials.
+
+1. Go to https://console.developers.google.com/apis/credentials
+
+2. Create a project
+
+3. Set up credentials
+
+   :authorized js origin: http://localhost:8000/
+   :callback url: http://localhost:8000/oauth2/signin/
+
+4. Open ``webapp-django/.env`` in an editor and add these lines::
+
+       OAUTH2_CLIENT_ID=<client id>
+       OAUTH2_CLIENT_SECRET=<secret>
+
+   where ``<client id>`` is the oauth client id and ``<secret>`` is the oauth
+   client secret that you got from step 3
+
+
+After that, Google signin should work.
```

Comment:
Are these instructions for setting up the oauth stuff right? Are there any other steps that are needed?